committee-steering/governance/sig-governance.md

```diff
@@ -1,60 +1,134 @@
-  enough *active* members to get a super-majority of votes cast, then removal may occur
-  through a [super-majority] vote between Chairs, Tech Leads and Subproject Owners.
-- Membership disagreements may be escalated to the SIG Chairs.  SIG Chair membership
-  disagreements may be escalated to the Steering Committee.
-- Members *MAY* decide to step down at anytime and propose a replacement.  Use lazy consensus amongst
-  other members with fallback on majority vote to accept proposal.  The candidate *SHOULD* be supported by a
-  majority of SIG Members or Subproject Contributors (as applicable).
-- Members *MAY* select additional members through a [super-majority] vote amongst members. This
-  *SHOULD* be supported by a majority of SIG Members or Subproject Contributors (as applicable).
+- Members of a role *SHOULD* remove any other members that have not communicated
+ a leave of absence and either cannot be reached for more than 1 month or are
+ not fulfilling their documented responsibilities for more than 1 month.
+  This may be done through a [super-majority] vote of members, or if there are
+  not enough *active* members to get a super-majority of votes cast, then
+  removal may occur through a [super-majority] vote between Chairs, Tech Leads
+  and Subproject Owners.
+
+#### Requirements
+- Members *MUST* be [community members] to be eligible to hold a leadership role
+  within a SIG.
+- SIGs *MAY* prefer various levels of domain knowledge depending on the
+role. This should be documented.  
+- People management interests - there's a lot of us!    
+
+#### Escalations
+- Membership disagreements *MAY* be escalated to the SIG Chairs. SIG Chair
+membership disagreements may be escalated to the Steering Committee.
+
+#### New members and Stepping Down
+- Members *MAY* decide to step down at anytime and propose a replacement.  Use
+lazy consensus amongst other members with fallback on majority vote to accept
+proposal.  The candidate *SHOULD* be supported by a majority of SIG Members or
+Subproject Contributors (as applicable).
+- Members *MAY* select additional members through a [super-majority] vote
+amongst members. This *SHOULD* be supported by a majority of SIG Members or
+Subproject Contributors (as applicable).
 
 ### Chair
 
 - Chair
-  - Run operations and processes governing the SIG
-  - Number: 2-3
-  - Membership tracked in [sigs.yaml]
+If no tech lead role is present, Chair assumes responsibilities from #tech-lead
+section in addition
+- Number: 2-3
+- Membership tracked in [sigs.yaml]  
+
+Run operations and processes governing the SIG:
+  - *SHOULD* define how priorities and commitments are managed and delegate to
+  other members as needed  
+  - *SHOULD* drive charter changes (including creation) to get community buy-in
+  but *MAY* delegate content creation to community members
+  - *SHOULD* identify, track, and maintain the groups features for current release
```

Comment:
I'm probably missing something but what does "groups feature" mean here?

Reply:
changing to SIGs enhancements - wdyt?